logging/google/cloud/logging/handlers/transports/background_thread.py

```diff
@@ -130,14 +132,14 @@ def _stop(self):
         self._stop_condition.release()
         self.stopped = True
 
-    def enqueue(self, record, message):
+    def enqueue(self, record, message, resource=_GLOBAL_RESOURCE):
```

Comment:
Feel like we're bringing in `_GLOBAL_RESOURCE` a lot of places, I'd probably just say make resource a required argument here, and then default to `_GLOBAL_RESOURCE` in `CloudLoggingHandler`,  and `AppEngineHandler` sets `gae_app` resource.